kolla/defaults.yml

```diff
@@ -30,13 +30,15 @@ blazar_project_enforcement_id: charge_code
 # When to send lease notification email
 blazar_minutes_before_end_lease: 2880
 blazar_default_max_lease_duration: 604800
+blazar_default_reservation_extension_window: 172800
 blazar_usage_default_allocated: 20000.0
 # Redis currently binds on public IP, use public FQDN
 blazar_usage_db_host: "{{ kolla_external_fqdn }}"
 blazar_usage_project_exemptions:
```

Comment:
These don't seem to be used anywhere. I think it needs to be in the `[enforcement]` config section.

Reply:
It is in enforcement:

```python
[enforcement]
default_max_lease_duration = {{ blazar_default_max_lease_duration }}	enabled_filters = MaxReservationLengthFilter, ExternalServiceFilter
project_max_lease_durations = {% for project_name in blazar_usage_project_exemptions %}{{ project_name }}:-1{% if not loop.last %},{% endif %}{% endfor %}	external_service_endpoint = {{ blazar_external_service_endpoint }}

max_reservation_length = {{ blazar_default_max_lease_duration }}
{# NOTE: newline above is required for correct config generation due to Jinja whitespace wackiness. Do not remove. #}	reservation_extension_window = {{ blazar_default_reservation_extension_window }}
```

Reply:
I was referring to the project whitelist -- where is that used in the updated Blazar config?

Reply:
Right right. I've set default to an empty list and updated site-configs.